utils/service_discovery/config_stores.py

```diff
@@ -0,0 +1,168 @@
+# std
+import logging
+import simplejson as json
+from os import path
+
+# 3p
+from urllib3.exceptions import TimeoutError
+
+# project
+from utils.checkfiles import get_check_class, get_auto_conf
+
+
+log = logging.getLogger(__name__)
+
+SD_TEMPLATE_DIR = '/datadog/check_configs'
+
+AUTO_CONF_IMAGES = {
```

Comment:
Maybe it should actually be in the yaml file directly ?
Something like
```yaml
docker_image:
   - redis
   - foo
init_config:

instances:
    - [...]
```

So we don't have to maintain this hardcoded list ? What do you think ?